src/frontend/components/DashboardDocument/index.tsx

```diff
@@ -148,6 +153,12 @@ const DashboardDocument = (props: DashboardDocumentProps) => {
           <Box direction={'row'}>
             <Box basis={'1/2'} margin={'small'}>
               {commonStatusLine}
+              <Box align={'center'} direction={'row'}>
+                <DashboardDocumentInternalHeading>
+                  {intl.formatMessage(messages.filename)}
+                </DashboardDocumentInternalHeading>
+                <div>{document.filename}</div>
+              </Box>
```

Comment:
Do you have a snapshot of rendered component?

Reply:
We don't use snapshot in the project.

Reply:
Sorry I though you talk about jest snapshot. 

![Screenshot from 2019-09-10 09-59-02](https://user-images.githubusercontent.com/767834/64595150-a99fb900-d3b1-11e9-878d-8420f260ea2f.png)

Reply:
I think we must work on the UI on another PR: the title on the right of the document icon is confusing (one would expect a file name?).